tmp/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
To get the dominant color of an image the image needs to be saved, so I save it into the tmp directory then delete it after the colors are good to go